Docs/source/Hardware/Boards.rst

```diff
@@ -68,23 +68,115 @@ A switching regulator (NCP1117) converts the +5V input to +3.3V. This does not n
+    * FAN 2 
+        * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+    * FAN 3 
+        * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+    * Fan 4 
+        * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+
+Schematic
+^^^^^^^^^
+*What does this circuit do?*
+    They control the fan's speed for cooling the battery pack. 
+*Why do we need it?*
+    This board makes sure the batteries don’t heat up by controlling the speed of the fans based on
+    the temperature. 
+*List of Circuit Components*
+    * Connector_Molex:Molex_Micro-Fit_3.0_43045-0612_2x03_P3.00mm_Vertical (LDRBRDConn) 
+        * Description: connects the leader board to fan board 
+        * Why is it necessary: so the fans can be supplied power 
+        * Justification for selection of specific part: this connector provides the right amount 
+          of power and connections for all fans being used 
+        * Associated passives/components:  
+            * 4 different BUK9M34-100EX 
+    * BUK9M34-100EX(4) 
+        * Description: a :term:`MOSFET <MOSFET>` that will switch the fans off and on 
+        * Why is it necessary: so the fans can be set to certain speeds by turning them on and off at varying rates
+        * Justification for selection of specific part: this part is used over others because of 
+          Q101 compliant, its suitable for thermally demanding environments, and true logic gate 
+          with VGS(th) rating of greather than 0.5V at 175°C 
+        * `Datasheet <https://assets.nexperia.com/documents/data-sheet/BUK9M34-100E.pdf>`__ 
+        * Associated passives/components:  
+            * 4 different diodes, Molex_MicroFit3.0_1x2xP3.00mm_PolarizingPeg_Vertical and the 
+              LDRBDConn above. 
+
+*List of Subsheet I/O*
+
+* Power +12 V 
+    * Input from BPS Leader Board 
+* Power GNDPWR 
+    * Input from BPS Leader Board 
+* FAN 1 
+    * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+* FAN 2 
+    * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+* FAN 3 
+    * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+* Fan 4 
+    * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+
+.. figure:: ../_static/FanBrdSch.png
+    :align: center
+
+    Fan Board Schematic
 
-Blocks
-    Mosfets - Each mosfet is between the power provided by the Master Board and power provided to the
-    fans. When the PWM signal is set, it switches the fan on or off at a certain duty cycle.
+**Dimensions: 45.00mm by 34.50mm**
 
-Connections
-    6 Connector: 4 of these connections are the PWM signals sent by the Master board. The other 2
-    are power and ground.
+Requirements/Constraints:  
+    We chose :term:`MOSFET <MOSFET>` body diodes to be used as the transistor to prevent voltage spikes from entering 
+    into the leader board (since the fan speeds can change and that could cause the spikes).
```

Comment:
Is this true? I thought we added D1-D4 as flyback diodes to stop voltage spikes caused by pulsing the fans. Correct me if I'm wrong @manthanand (I think you designed this board)

Reply:
Yea, I told them to add that but I meant to say flyback diode.